examples/website-demos/social-messaging/src/components/ChatContainer/ChatContainer.tsx

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from 'react';
+
+import { ChannelSort, LiteralStringForUnion, StreamChat } from 'stream-chat';
+import { Channel, ChannelList, Chat } from 'stream-chat-react';
+
+import { ChannelContainer } from '../ChannelContainer/ChannelContainer';
+import { SideDrawer } from '../SideDrawer/SideDrawer';
+import { SocialChannelList } from '../SocialChannelList/SocialChannelList';
+import { SocialChannelListHeader } from '../SocialChannelList/SocialChannelListHeader';
+import { SocialChannelPreview } from '../ChannelPreview/SocialChannelPreview';
+import { SocialMessageInput } from '../MessageInput/SocialMessageInput';
+
+import { useViewContext } from '../../contexts/ViewContext';
+
+import './ChatContainer.scss';
+
+const apiKey = process.env.REACT_APP_STREAM_KEY;
+const user = process.env.REACT_APP_USER_ID;
+const userImage = process.env.USER_IMAGE;
+const userToken = process.env.REACT_APP_USER_TOKEN;
+
+const userToConnect: { id: string; name?: string; image?: string } = {
+  id: user!,
+  name: user!,
+  image: userImage,
+};
+
+const filters = { type: 'messaging', members: { $in: [user!] } };
+const options = { message_limit: 20 };
```

Comment:
let's make limit larger than default, not smaller, to reduce load more renders, maybe call it 30